imap/conversations.c

```diff
@@ -709,9 +709,9 @@ EXPORTED void conversation_normalise_subject(struct buf *s)
     if (!initialised_res) {
         r = regcomp(&whitespace_re, "([ \t\r\n]+|\xC2\xA0)", REG_EXTENDED);
         assert(r == 0);
-        r = regcomp(&bracket_re, "(\\[[^\\[\\]]*\\])|(^[^\\[]*\\])|(\\[[^\\]]*$)", REG_EXTENDED);
+        r = regcomp(&bracket_re, "(\\[[^]\\[]*])|(^[^\\[]*])|(\\[[^]]*$)", REG_EXTENDED);
         assert(r == 0);
-        r = regcomp(&relike_token_re, "^[ \\t]*[^\\s]+:", REG_EXTENDED);
+        r = regcomp(&relike_token_re, "^[ \\t]*[^\\t\\r\\n\\f]+:", REG_EXTENDED);
         assert(r == 0);
```

Comment:
Doesn't \s include SP as well, or don't we want to exclude SP here?

Reply:
Otherwise, this appears to solve the problem

Reply:
Thanks. I pushed a fix in https://github.com/cyrusimap/cyrus-imapd/pull/3110/commits/2df28618c871d17eea3c7dcf37f74dcdc2f03f73